### ISSUE TITLE
Synapserutils conflict

### ISSUE THREAD

**Issue by allaway** [Sage-Bionetworks/challengerutils#18]
**Describe the bug**
In addition to `synapser` there is also a conflict with `synapserutils` - this and the workaround should be noted and added to the docs.

**Comment by thomasyu888** [comment]
Ah, are you using `synapserutils` as well for something?

**Comment by allaway** [comment]
No, I was just being lazy and was testing some synapserutils function for a collaborator, but I loaded it into a session that had `challengerutils` active. 

It's probably not a huge deal, but worth adding to the docs in case people run into it!
